circom/aes256_gcm_test.go

```diff
@@ -152,65 +152,47 @@ func TestAESGCM256Circuit(t *testing.T) {
 		py,
 	}
 	RawKey := Pub.RawBytes()
-	//m := RawKey[:]
+	m := RawKey[:]
+	//m := []byte{0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0A, 0x0B, 0x0C, 0x0D, 0x0E, 0x0F, 0x01}
 	//m := []byte{0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0A, 0x0B, 0x0C, 0x0D}
 	//m := []byte{0x32, 0x43, 0xf6, 0xa8, 0x88, 0x5a, 0x30, 0x8d, 0x31, 0x31, 0x98, 0xa2, 0xe0, 0x37, 0x07, 0x34}
-	m := []byte{0x32, 0x43, 0xf6, 0xa8, 0x88, 0x5a, 0x30, 0x8d, 0x31, 0x31, 0x98, 0xa2, 0xe0, 0x37, 0x07, 0x34, 0x32, 0x43, 0xf6, 0xa8, 0x88, 0x5a, 0x30, 0x8d, 0x31, 0x31, 0x98, 0xa2, 0xe0, 0x37, 0x07, 0x34}
-	M_bytes := make([]frontend.Variable, len(m))
+	//m := []byte{0x32, 0x43, 0xf6, 0xa8, 0x88, 0x5a, 0x30, 0x8d, 0x31, 0x31, 0x98, 0xa2, 0xe0, 0x37, 0x07, 0x34, 0x32, 0x43, 0xf6, 0xa8, 0x88, 0x5a, 0x30, 0x8d, 0x31, 0x31, 0x98, 0xa2, 0xe0, 0x37, 0x07, 0x34}
+	M_bytes := make([]uints.U8, len(m))
 	for i := 0; i < len(m); i++ {
-		M_bytes[i] = m[i]
+		M_bytes[i] = uints.U8{Val: m[i]}
 	}
 
 	hasher := sha3.New256()
 	hasher.Write(RawKey[:])
-	//expected := hasher.Sum(nil)
-	expected := [32]byte{0x60, 0x3d, 0xeb, 0x10, 0x15, 0xca, 0x71, 0xbe, 0x2b, 0x73, 0xae, 0xf0, 0x85, 0x7d, 0x77, 0x81, 0x1f, 0x35, 0x2c, 0x07, 0x3b, 0x61, 0x08, 0xd7, 0x2d, 0x98, 0x10, 0xa3, 0x09, 0x14, 0xdf, 0xf4}
-	keyBytes := [32]frontend.Variable{}
+	expected := hasher.Sum(nil)
+	//expected := [32]byte{0x60, 0x3d, 0xeb, 0x10, 0x15, 0xca, 0x71, 0xbe, 0x2b, 0x73, 0xae, 0xf0, 0x85, 0x7d, 0x77, 0x81, 0x1f, 0x35, 0x2c, 0x07, 0x3b, 0x61, 0x08, 0xd7, 0x2d, 0x98, 0x10, 0xa3, 0x09, 0x14, 0xdf, 0xf4}
```

Comment:
Does this test work correctly?

Reply:
<img width="626" alt="1728960444364" src="https://github.com/user-attachments/assets/58429faf-7c04-4b4d-a6e2-95490f4f1bd0">

Reply:
Is your execution environment running incorrectly?